test/easy_ssl_test.exs

```diff
@@ -23,7 +23,7 @@ defmodule EasySSLTest do
       |> Enum.each(fn cert_filename ->
             original_cert = File.read!(@der_cert_dir <> cert_filename)
               |> EasySSL.parse_der
-
+            IO.inspect original_cert
```

Comment:
Just FYI I'm going to merge this but remove this later before I release the next version.

Reply:
Sorry about that! Thanks for merging this.